shell/platform/darwin/ios/framework/Source/FlutterDartProjectTest.mm

```diff
@@ -73,15 +73,54 @@ - (void)testFLTFrameworkBundleInternalWhenBundleIsPresent {
   XCTAssertNotNil(found);
 }
 
+- (void)testDisableImpellerSettingIsCorrectlyParsed {
+  id mockMainBundle = OCMPartialMock([NSBundle mainBundle]);
+  OCMStub([mockMainBundle objectForInfoDictionaryKey:@"FLTEnableImpeller"]).andReturn(@"NO");
+
+  auto settings = FLTDefaultSettingsForBundle();
+  // Check settings.enable_impeller value is same as the value defined in Info.plist.
+  XCTAssertEqual(settings.enable_impeller, NO);
```

Comment:
`XCTAssertTrue` and `XCTAssertFalse` are a little more idiomatic, but it's a minor nit.
```suggestion
  XCTAssertFalse(settings.enable_impeller);
```

Reply:
I suspect I'll be touching this file again soon, and I'll keep this in mind for a cleanup then.